### DISCUSSION TITLE
Admin user locked out. How to update the admin user's status?

### DISCUSSION THREAD

**Discussion by arladmin** [directus/directus#7276] [Archive]
### Preflight Checklist

- [X] I have completed all [Troubleshooting Steps](https://docs.directus.io/getting-started/support/#troubleshooting-steps).
- [X] I'm on [the latest version of Directus](https://github.com/directus/directus/releases).
- [X] There's [no other issue](https://github.com/directus/directus/issues) that already describes my problem.

### Describe the Bug

Out of seemingly nowhere, the admin user for my directus instance has become suspended, around 30 mins. after the last usage.

Note: 
- There des exist a strict password policy (retry attempts: 2). 
- MFA is currently disabled

But even going the Forgot Password route isn't working. I have been able to change the password using the reset link, but the system still doesn't allow me to login!

**How do i regain access?** (I do have control over my server)

Also, how to figure out the cause, and then prevent such an event from reoccurring?
<hr>

**Update:** 

I've been able to regain access by:
- resetting the password from the Forgot Password link
- then modifying the admin user's status to active, by directly modifying the database

But, is this the only way to regain access, if something like this occurs again?
Is there any best-practice/fail-safe to avoid this sort of an event?
(Have already switched on 2FA as well as generated a separate admin account for server-to-server API access)

<hr>

### To Reproduce

Same as above.

### What version of Directus are you using?

v9.0.0-rc.88

### What version of Node.js are you using?

16.5.0

### What database are you using?

Postgres 12

### What browser are you using?

Chrome

### What operating system are you using?

Windows

### How are you deploying Directus?

Docker

**Comment by irishburlybear** [comment]
This is happening to me also, Every (hour-ish) My admins get locked out. No error really given. Luckily I'm usually logged in so I can change the password back. I found if you are locked out... passwords are generated through Argon2. I used this website( https://argon2.online/ ) to generate and match the settings in the password field. I'm using phpmyadmin to copy the password back.

**Comment by benhaynes** [comment]
If login attempts is only set to two, then this makes sense. If you hit that number, the account status will shift to "suspended", and a different admin user will need to correct the status to allow login (or a direct database update is needed).

We could decide if going through the forgot password flow should reset your account status. However, if an admin were to suspend your account, forgot password should not undo that... so we'll have to take that into consideration.

**Comment by arladmin** [comment]
Hi @benhaynes , i was thinking, maybe the super admin account could have the capability to never get its account suspended?

**Comment by irishburlybear** [comment]
Oh I should note. I have auth login attempts set at 900000000.

**Comment by benhaynes** [comment]
> Oh I should note. I have auth login attempts set at 900000000.

Your issue seems to be different/unrelated. The OP is describing a "lock out" caused by the User Status being set to `suspended`... but you're experiencing a password change?

**Reply by b4d4** [reply]
Hi, I'm experiencing a password change. And i need to reset password in order to login. Is there any expiration time for the password?. I just checked argon hash from db and the verification failed, how its possible if i didnt change the pass?

**Reply by joselcvarela** [reply]
We are not changing password in any manner so, maybe another user with access to that account have changed the password?

**Comment by rijkvanzanten** [comment]
> Out of seemingly nowhere, the admin user for my directus instance has become suspended

The "Auth Login Attempts" feature allows you to set after how many failed login attempts the user will become suspended. If you set this to a low value like 2, 2 consecutive failed login attempts will suspend the user.

> How do i regain access? (I do have control over my server)

Simply set the `status` for the user back to `active`, either directly in the database under `directus_users`, or by using another user that has update access to Users 👍🏻

**Comment by bhagya1101** [comment]
Isn't there any way we can know if the user is suspended because of consecutive login attempts? Like in activity logs

**Reply by thvdbk** [reply]
clearly something that would be useful here for traceability purpose. It's a modification of user by system, could it be logged in activity / revisions ?
